projects/ngx-d3timeline/src/lib/ngx-d3timeline.component.ts

```diff
@@ -53,10 +56,18 @@ export class NgxD3timelineComponent implements AfterViewInit {
     this.timeline.setTimeOrientation(value);
   }
 
+  @Output() dropped = new EventEmitter<Activity>();
```

Comment:
maybe a bit more meaningful?

```suggestion
  @Output() activityDropped = new EventEmitter<Activity>();
```

Reply:
done